readme-template.md

```diff
@@ -207,30 +209,54 @@ A thread can also be given the moderation options when joining. You can pass `fi
 const thread = await space.joinThread('myThread', { firstModerator: 'some3ID', members: true })
 ```
 
-Lastly a thread can be joined by its address.
+##### 1.b Creating a Confidential Thread
+
+To create and join a confidential thread.
+
+```js
+const thread = await space.createConfidentialThread('myConfThread')
+```
+
+At creation you will likely want to add other members so that they can read and write messages to the thread, as shown below.
+
+##### 2. Joining a Thread
+
+An existing public or confidential thread can be joined by its address. Confidential threads are best referenced by their address.
```

Comment:
Add something about how you get the address of a thread.

Reply:
thats right below

Reply:
haha 🙈